tests/supervisor/test_supervisor_utils.c

```diff
@@ -0,0 +1,58 @@
+#define _GNU_SOURCE
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <setjmp.h>
+#include <stdint.h>
+#include <cmocka.h>
+
+#include "utils/log.h"
+#include "supervisor/supervisor_utils.h"
+
+static const UT_icd config_ifinfo_icd = {sizeof(config_ifinfo_t), NULL, NULL,
+                                         NULL};
+
+static void test_allocate_vlan(void **state) {
+  (void)state; /* unused */
+  uint8_t mac_addr[6] = {0x04, 0xf0, 0x21, 0x5a, 0xf4, 0xc4};
+
+  struct supervisor_context ctx = {};
```

Comment:
The ` = {}` syntax is only valid in C23, so it might not work on older compilers.

See https://en.cppreference.com/w/c/language/initialization

```suggestion
  struct supervisor_context ctx = {};
```